buildpipeline/windows.groovy

```diff
@@ -35,9 +35,6 @@ simpleNode('Windows_NT','latest') {
             bat 'type init-tools.log'
```

Comment:
While you are in here could you also remove this line as I left it there for some debugging that isn't necessary any longer.